draft-ietf-quic-ack-frequency.md

```diff
@@ -139,7 +139,8 @@ endpoint performance in the following ways:
   critical for high bandwidth connections.
 
 - Similarly, receiving and processing UDP packets can also be CPU intensive, and
-  reducing acknowledgment frequency reduces this cost at a data sender.
+  reducing number of packets that only contain acknowledgments also reduces
+  this cost at a acknowledgments receiver which is the data sender.
```

Comment:
```suggestion
  reducing the number of acknowledgments also reduces the processing
  cost on the sender side.
```

Reply:
I thinks this change mostly reverts my proposed change. I think the original sentence was correct, so if people think more words don't help, we don't have to accept this PR. But I thought it might clarify things. I guess it would be good if @LPardue takes a look who raised the issue.

Reply:
So after thinking some more, IMO there's a bit too much passive voice in the current document, making it hard to determine the subject and the effects. I made a new PR that makes things clearer to me at least.